pkg/resources/manager/functions.go

```diff
@@ -111,7 +119,7 @@ func GetMachineClassRequiredResources(name string) (core.ResourceList, error) {
 		return nil, err
 	}
 
-	return class.resources.DeepCopy(), nil
+	return class.Capabilities.DeepCopy(), nil
```

Comment:
i use capabilities for keep compatibility with https://github.com/ironcore-dev/ironcore/blob/main/api/compute/v1alpha1/machineclass_types.go#L25

Technically for us it is still resourcelist with resources cannot it be more confusing? When i rename it, it can create confusing behavior because iri.MachineClass has capabilities too which are different.

Reply:
discussed offline